parsing/parser.mly

```diff
@@ -1616,7 +1615,7 @@ labeled_simple_pattern:
       { (Nolabel, None, $1) }
 ;
 
-pattern_var:
+%inline pattern_var:
```

Comment:
`pattern_var_(lin) -> pattern_var(nonbr)` suggests inlining `pattern_var_` instead.

Reply:
(Or literally inlining `pattern_var_`'s definition and removing the symbol.)